src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs

```diff
@@ -0,0 +1,94 @@
+(ns org.broadinstitute.firecloud-ui.page.library.library_page
+  (:require
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common :as common]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.icons :as icons]
+    [org.broadinstitute.firecloud-ui.common.modal :as modal]
+    [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.common.table :as table]
+    [org.broadinstitute.firecloud-ui.common.table-utils :refer [float-right]]
+    [org.broadinstitute.firecloud-ui.nav :as nav]
+    [org.broadinstitute.firecloud-ui.common.table-utils :as table-utils]
+    [org.broadinstitute.firecloud-ui.utils :as utils]
+    ))
+
+
+(react/defc DatasetsTable
+  {:render
+   (fn [{:keys [state this]}]
+     (cond
+       (:error-message @state) (style/create-server-error-message (:error-message @state))
+       :else
+       [:div {:style {:flex "2 0 auto" :marginRight "1em"}}
+        [:div {:style {:fontWeight 700 :fontSize "125%"}} "Search Results: "
+         [:span {:style {:fontWeight 100}} (str (:total @state) " Datasets found")]]
```

Comment:
This means that while the data is loading, it displays "Search Results: Datasets found".  I'd suggest wrapping this span with some code that checks if `(:total @state)` exists and rendering "pending..." when it's still loading.